### PR TITLE
Document team-events endpoint

### PR DESCRIPTION
**Pull request by olivierdagenais**
This branch provides a very basic landing page that documents the available events, with sample request payloads.

Manual testing
--------------

1. Launched the test Jenkins instance on my workstation to port 9090.
2. Pointed my web browser to `http://localhost:9090/team-events/` and saw something like this:
image
3. Created the `payload.json` file and copy-pasted the example payload into it.
4. Copy-pasted the `curl` command-line in my console window.
5. The output was exactly as shown in the screenshot.

Mission accomplished!

### CODE REVIEW

File: src/main/java/hudson/plugins/tfs/model/GitCodePushedEventArgs.java [hunk 1]
```diff
@@ -13,6 +13,266 @@
...
+        "            \"System.ChangedDate\": \"2014-12-29T20:49:20.77Z\",\n" +
+        "            \"System.ChangedBy\": \"Jamal Hartnett <fabrikamfiber4@hotmail.com>\",\n" +
+        "            \"System.AuthorizedAs\": \"Jamal Hartnett <fabrikamfiber4@hotmail.com>\",\n" +
+        "            \"System.PersonId\": 77331,\n" +
+        "            \"System.Watermark\": 607,\n" +
+        "            \"System.Title\": \"Customer can sign in using their Microsoft Account\",\n" +
+        "            \"Microsoft.VSTS.Scheduling.Effort\": 8,\n" +
+        "            \"WEF_6CB513B6E70E43499D9FC94E5BBFB784_System.ExtensionMarker\": true,\n" +
+        "            \"WEF_6CB513B6E70E43499D9FC94E5BBFB784_Kanban.Column\": \"New\",\n" +
+        "            \"System.Description\": \"Our authorization logic needs to allow for users with Microsoft accounts (formerly Live Ids) - http://msdn.microsoft.com/en-us/library/live/hh826547.aspx\"\n" +
```
Comment (DavidStaheli) [inline src/main/java/hudson/plugins/tfs/model/GitCodePushedEventArgs.java]: "Microsoft accounts" → "Microsoft Accounts"
"Live Ids" → "Windows Live IDs"

File: src/main/java/hudson/plugins/tfs/model/PullRequestMergeCommitCreatedEventArgs.java [hunk 2]
```diff
@@ -9,6 +9,267 @@
...
+        "            \"System.ChangedDate\": \"2014-12-29T20:49:20.77Z\",\n" +
+        "            \"System.ChangedBy\": \"Jamal Hartnett <fabrikamfiber4@hotmail.com>\",\n" +
+        "            \"System.AuthorizedAs\": \"Jamal Hartnett <fabrikamfiber4@hotmail.com>\",\n" +
+        "            \"System.PersonId\": 77331,\n" +
+        "            \"System.Watermark\": 607,\n" +
+        "            \"System.Title\": \"Customer can sign in using their Microsoft Account\",\n" +
+        "            \"Microsoft.VSTS.Scheduling.Effort\": 8,\n" +
+        "            \"WEF_6CB513B6E70E43499D9FC94E5BBFB784_System.ExtensionMarker\": true,\n" +
+        "            \"WEF_6CB513B6E70E43499D9FC94E5BBFB784_Kanban.Column\": \"New\",\n" +
+        "            \"System.Description\": \"Our authorization logic needs to allow for users with Microsoft accounts (formerly Live Ids) - http://msdn.microsoft.com/en-us/library/live/hh826547.aspx\"\n" +
```
Comment (DavidStaheli) [inline src/main/java/hudson/plugins/tfs/model/PullRequestMergeCommitCreatedEventArgs.java]: "Microsoft accounts" → "Microsoft Accounts"
"Live Ids" → "Windows Live IDs"

**Comment by DavidStaheli** [comment]
Looks good!  I left 2 minor comments.

File: src/main/java/hudson/plugins/tfs/model/PullRequestMergeCommitCreatedEventArgs.java (hunk 2 above)
Comment (olivierdagenais) [inline src/main/java/hudson/plugins/tfs/model/PullRequestMergeCommitCreatedEventArgs.java]: Huh, good catch!  I just copy/pasted the _sample response_ from [Get a list of work items > By IDs](https://www.visualstudio.com/en-us/docs/integrate/api/wit/work-items#by-ids).  Can you relay your suggestions to them, too?

**Comment by olivierdagenais** [comment]
Rebased and then applied suggestions in 73f8fd7dd8f98a6ae814f317bb02536ae5a481f1.

Thanks for the code review!

**Pull request merged**